nautobot/docs/user-guide/administration/guides/prometheus-metrics.md

```diff
@@ -27,6 +27,19 @@ CACHES = {
 
 For more information see the [django-prometheus](https://github.com/korfuri/django-prometheus) docs.
 
+## Authentication
+
```

Comment:
Since this is a new feature, would it be more appropriate to target the PR to `next` (i.e. v2.2.0) instead of `develop`? Either way, let's add a version annotation here, i.e. either:

```
+++ 2.1.3
```

or 

```
+++ 2.2.0
```

Reply:
After some discussion we're fine with this going into `develop` at this time.

Reply:
Let's use 2.1.4. Thanks!

Reply:
That's been added

Reply:
updated to 2.1.5